src/main/java/org/opentripplanner/middleware/models/BugsnagEvent.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.middleware.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.opentripplanner.middleware.bugsnag.response.App;
+import org.opentripplanner.middleware.bugsnag.response.EventException;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents a Bugsnag error events. Class is used for both Mongo storage and JSON deserializing
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BugsnagEvent extends Model {
+
+    @JsonProperty("id")
+    public String eventDataId;
```

Comment:
Add JSDoc to the fields.

Reply:
If these are all direct copies of what's in Bugsnag, then we could probably just note that in the class JavaDoc and add a link to the docs.

Reply:
Have updated to include reference.